scripts/test.sh

```diff
@@ -14,4 +14,4 @@ done
 cur_dir=$(dirname "$0")
 ${cur_dir}/build.sh
 
-docker run -e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} -e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} -v /tmp:/tmp -v ${PWD}/:/usr/local/app --entrypoint pytest globalforestwatch/pixetl  --cov-report term --cov-report xml --cov=gfw_pixetl tests/  # pragma: allowlist secret
+docker run -e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} -e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} -v /tmp:/tmp -v ${PWD}/:/usr/local/app --entrypoint pytest globalforestwatch/pixetl  --cov-report term --cov-report xml --cov=. tests/  # pragma: allowlist secret
```

Comment:
It took me quite a while to figure out on how to set this up correctly.
To allow integration with Code Climate, we have to make sure to write coverage report to XML file. Mount CWD to app directory in docker container, to be able to access xml outside docker container. 
There also seems to be a bug with the Code Climate test-coverage binary. I have to run the coverage test for the entire rep `--cov=. not just the `gfw_pixetl` folder. This way, the test modules are also includes in the coverage report, which increases the coverage reported, but at least it works for now.
Results should show up, once posted to master

Reply:
This should should fix #9